tracee-ebpf/tracee/events_definitions.go

```diff
@@ -5722,6 +5722,7 @@ var EventsDefinitions = map[int32]EventDefinition{
 		Sets:           []string{"default", "proc", "proc_life"},
 		Params: []external.ArgMeta{
 			{Type: "long", Name: "exit_code"},
+			{Type: "int", Name: "is_process_group_exit"},
```

Comment:
Let's use Linux terminology, which is also shorter - group_dead

Reply:
Also, why not using bool?

Reply:
So I tried using bool but I got a wrong value.
I will try to look if I can do it.